### DISCUSSION TITLE
Actividad 1 - Grupo 5

### DISCUSSION THREAD

**Discussion by DanielUTN** [frlp-utn-ingsoft/catedra#97] [Show and tell]
https://github.com/DanielUTN/vetsoft

**Comment by ivanduva** [comment]
Buenas,
La actividad tiene algunos problemas menores que les pedimos que solucionen para aprobar:

- El form de editar medicamento se rompe al ingresar float o caracteres no numéricos en el campo dosis, en lugar de hacer la validación
- Lo mismo en el caso de producto, pero este casi si permite ingresar un float, lo cual está bien. Se rompe en el caso de agregar cualquier otro caracter.
- En el form de veterinario, el campo telefono permite ingresar cualquier caracter cuando deberían ser solo números

Tienen tiempo para solucionarlo hasta la fecha de entrega de la próxima actividad de test que la informaremos la próxima clase.

Además les dejamos comentarios para las próximas actividades que deberían trabajar como grupo:

- Hay un muy buen intento de estandarizar los commits, pero no todos lo siguen. Estaría bueno que todos tengan el mismo formato. También en algunos casos vemos muchas menciones a archivos y funciones en los mensajes de commit, lo cual es redundante porque el commit ya contiene esa información. El mensaje debería ser una síntesis de la funcionalidad que resuelve el commit. Tampoco es necesario justificar decisiones ni hacer aclaraciones de si afecta el funcionamiento o no.
- Los nombres de branches no son relevantes. Deberían ser relativos a la funcionalidad, sin hacer referencia al usuario que creó el branch.

Más allá de esto, la actividad está bien y valoramos que hayan realizado la relación mascota-cliente que era opcional.
Saludos!

**Reply by JoseLlontop** [reply]
Buenas, ya realizamos las correcciones.

**Reply by ivanduva** [reply]
Bien, queda aprobado entonces.
Saludos
